src/api/NotesApi.ts

```diff
@@ -84,6 +66,35 @@ export class NotesApi implements INotesApi {
         }
     }
 
+    async fetchAllNotes(): Promise<INote[]> {
+        try {
+            let notes: SPNote[] = await this.notesList.items.select("Id", "Request/Id", "Title", "Text", "Modified", "Author/Id", "Author/Title", "Author/EMail", "Status").expand("Request", "Author").get();
```

Comment:
This is going to get **_all_** notes right? I'm a little concerned with performance here. Wondering if there should be  an overloaded function, something like
`fetchNotesByRequestId(requests: number[])`
that would take an array of current items and send a batched request to SP to get the relevant items only